config/webpack/environment.js

```diff
@@ -15,9 +15,15 @@
  */
 
 const { environment } = require('@rails/webpacker');
-const { VueLoaderPlugin } = require('vue-loader')
-const vue = require('./loaders/vue')
+const { VueLoaderPlugin } = require('vue-loader');
+const vue = require('./loaders/vue');
+const webpack = require('webpack');
 
+
+environment.plugins.prepend('jQuery', new webpack.ProvidePlugin({
+  jQuery: 'jquery',
+  $: 'jquery',
+}));
```

Comment:
What kind of blast radius does this change have?

Reply:
As far as I can tell... just "adds jQuery"